ui/jobs/jobs.css

```diff
@@ -1141,3 +1141,14 @@
 .blu-color-lucid {
   background-color: rgb(227, 148, 210);
 }
+
+.proc-box[incombat] .flash-when-expired.expired {
+  display: block;
+  animation: flash 1s infinite;
```

Comment:
I don't think `infinite` is a good idea here, even it is only limited in combat.
To think when we have a boss who got a long long cutscene, for example, e1n, an infinite loop flash might be painful.
I suggest that `10` times or `8` times here is ok.

Reply:
I didn't found this to be a problem, however I see your point, what about make this configurable with options:
6/8/10/infinite?

Reply:
what about a number textbox? and 0 means infinite.